src/.env.local.js.example

```diff
@@ -39,4 +39,16 @@ module.exports = {
   pageTitle: null,
   // OPTIONAL: The hostname to emulate when testing.
   hostname: "example.local",
+  // OPTIONAL: Provides config for adding autocomplete functionality to text search
+  // autocomplete : {
+  //   url: <your-endpoint-for-autocomplete-results> // required @todo document accepted types
```

Comment:
I had problems because there are not commas after the samples. We should have commas after each element.

Reply:
Sorry about that.  Updated in https://github.com/palantirnet/federated-search-react/pull/26/commits/e1f663c5a1397e448aafa2d2b404498abe7067ef